tests/foreman/cli/test_docker.py

```diff
@@ -71,7 +71,7 @@ def test_bugzilla_1190122(self):
         # Some images do not have tags associated with it, ignore those because
         # we want to check the tag information
         images = [
-            image for image in result.stdout if int(image['tags']) > 0
+            image for image in result.stdout if int(image['tag-count']) > 0
```

Comment:
```image['tags']``` is a string containing all tags, separated by comma. so we shall check either by ```int(image['tag-count'])``` or by ```len(image['tags'])```.